src/sentry/api/endpoints/organization_details.py

```diff
@@ -516,6 +522,35 @@ def put(self, request, organization):
                     data=changed_data,
                 )
 
+                # Temporarily writing org-level apdex changes to ProjectTransactionThreshold
+                # for orgs who don't have the feature enabled so that when this
+                # feature is GA'ed it captures the orgs current apdex threshold.
+                if "apdexThreshold" in changed_data and not features.has(
+                    "organizations:project-transaction-threshold", organization
+                ):
+                    match = re.match(r".*to (\d+)$", changed_data["apdexThreshold"])
```

Comment:
You should be able to use `serializer.validated_data.get("apdexThreshold")` instead of regexing on the text.